src/styles/colors.ts

```diff
@@ -27,6 +28,7 @@ export const COLOR_GRAY_TRANSPARENT_3 = 'rgba(82, 95, 127, 0.3)';
 
 // text colors
 export const COLOR_DARK_1 = 'rgb(8, 28, 63)';
+export const COLOR_DARK_2 = 'rgb(87,101,116)';
```

Comment:
The value color for the `COLOR_DARK_2` const is equal to the hexadecimal  `COLOR_GRAY_4` const, I think that instead create a new const we should change the `#576574` by `rgb(87,101,116)`